aries-site/src/examples/components/datatable/DataTableVerticalListExample.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button, 
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import ServerGroups from '../../../data/mockData/serverGroups.json';
+
+/* This function displays an arr's values in a vertical list within a table 
+   cell, hiding all values from view that exceed the given maximum list 
+   length until the "View More" button is clicked, at which point the 
+   button label changes to "View Less". */
+function ViewMore(arr, maxParam) {
+    const [viewMoreBool, setViewMoreBool] = useState(false);
```

Comment:
```suggestion
    const [viewMore, setViewMore] = useState(false);
```
Typically we just keep a variable like this named more simply. This will need to be adjusted in the instances of `viewMoreBool` below.